server/services/voip/service.ts

```diff
@@ -84,7 +85,34 @@ export class VoipService extends ServiceClass implements IVoipService {
 	}
 
 	async getServerConfigData(type: ServerType): Promise<IVoipServerConfig | null> {
-		return this.VoipServerConfiguration.findOne({ type, active: true });
+		// TODO: Decide the approach we will take regarding settings after the MVP,
+		// For now this work around should be enough.
+
+		// const config = this.VoipServerConfiguration.findOne({ type, active: true });
+
+		const management = type === ServerType.MANAGEMENT;
+		return {
+			_id: '',
```

Comment:
looks like `IVoipServerConfig` doesn't need extend `IRocketChatRecord` anymore.. so you don't need to provide `_id: ''` (which doesn't make sense anyways)